### PR TITLE
Add People API support

### LINKED ISSUE #559: Missing clasp support for People API
## Expected Behavior
People API can be enabled via the web UI. The API should be on when adding it to `appsscript.json` like this
`...
"dependencies": {
    "enabledAdvancedServices": [      {
        "userSymbol": "People",
        "serviceId": "people",
        "version": "v1"
      }
    ]
  }
...`

## Actual Behavior
After `clasp push` PEOPLE API is switched off. Looks like this happens because the `people` constant is not added to the `PUBLIC_ADVANCED_SERVICES` (https://github.com/google/clasp/blob/f00bbbf019befd79e5d82790c5647bdfb82c534c/src/apis.ts#L38)

## Steps to Reproduce the Problem

1. In appscript.json add a snippet from above 
2. `clasp push`
3. In project's web UI -> Resources -> Advanced Google services... 
4. People API is OFF

## Specifications

- Node version (`node -v`): v10.14.2
- Version (`clasp -v`): 1.6.3
- OS (Mac/Linux/Windows): Windows

### PR DESCRIPTION
**Pull request by tawAsh1**
Fixes #559  (it's a good idea to open an issue first for discussion)

- [x] `npm run test` succeeds.
- [x] `npm run lint` succeeds.
- [x] Appropriate changes to README are included in PR.

Add People API description to AdvancedService[]

### CODE REVIEW

File: src/apis.ts [hunk 1]
```diff
@@ -171,6 +171,11 @@ export const PUBLIC_ADVANCED_SERVICES: AdvancedService[] = [
     serviceId: 'analytics',
     version: 'v3',
   },
+  {
+    userSymbol: 'People',
+    serviceId: 'peopleapi',
```
Comment (erickoledadevrel) [inline src/apis.ts]: Are you sure "peopleapi" is correct? Looking at the Service Usage API, it seems like it should just be "people".

Comment (tawAsh1) [reply src/apis.ts]: You are right. When I use Service Usage API, it says serviceId is `people`.

However, If I clone the script with PeopleAPI enabled on the GUI, appscript.json says `serviceId:" peopleapi "`.

Comment (erickoledadevrel) [reply src/apis.ts]: Ah, you are right. This seems like another case of issue #512. It was assumed that the `serviceId` in  `appsscript.json` would be the same as the prefix used by the Service Usage API, but that isn't always the case. Until that issue is fixed we won't be able to support the People API fully.

**Comment by grant** [comment]
Do you want to close this PR until #512 is fixed?

**Pull request closed without merging**